### PR TITLE
Add compatible registries page

### PR DESCRIPTION
**Pull request by carolynvs**
Move content from https://porter.sh/compatible-registries over to https://cnab.io/registries.

Link to registries from homepage
Screen Shot 2020-05-21 at 10 38 45 AM

Preview at https://deploy-preview-37--cnabio.netlify.app/registries/

Please review if it's okay for me to use the logo's for the registries that we have verified. Docker's (moby) logo came from their official logos page. I checked that ACR's logo is our latest version of the logo, same for GCR as well.

### CODE REVIEW

**Comment by carolynvs** [comment]
@chris-crone If you could find out which is the appropriate logo to use when representing Docker Hub, that would be great. I picked the moby logo from this page below since the others were for the company, and I thought this one was the best representation of the registry? The logo used on hub isn't available on the media page so I wasn't sure what to use. 😀 

https://www.docker.com/company/newsroom/media-resources

**Comment by chris-crone** [comment]
That site is the one to use, you should use either "Docker Logo Horizontal Blue" or "Docker Logo Blue" depending on where you're putting it.

File: content/registries.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+| -------- | --------------- |
+| **Azure Container Registry (ACR)** | **Yes** |
+| Artifactory | No |
+| **Docker Hub** | **Yes** |
+| **DigitalOcean Container Registry** | **Yes** |
+| Amazon Elastic Container Registry (ECR) | No |
+| **Google Cloud Registry (GCR)** | **Yes** |  
+| GitHub Packages | No |
+| Nexus | No |
+| Quay | No |
```
Comment (silvin-lubecki) [inline content/registries.md:26]: Question: Should we add also Harbor 2, since they added CNAB support [officially](https://goharbor.io/blog/harbor-2.0/)?

Comment (carolynvs) [reply content/registries.md:26]: Of course! I'll add that today

**Comment by carolynvs** [comment]
Thanks! I went with the blue docker logo to fit in with the layout. 

Screen Shot 2020-05-26 at 8 17 37 AM

@chris-crone PTAL

**Comment by chris-crone** [comment]
LGTM, thanks @carolynvs!

**Pull request merged**